docs/drawer-based-navigation.md

```diff
@@ -3,6 +3,7 @@ id: drawer-based-navigation
 title: Drawer navigation
 sidebar_label: Drawer navigation
 ---
+This guide covers [createDrawerNavigator](https://reactnavigation.org/docs/en/drawer-navigator.html). 
```

Comment:
paths should be relative, so: `[createDrawerNavigator](drawer-navigator.html)`. same in the 3.x doc

Reply:
:+1:!